Cargo.toml

```diff
@@ -86,7 +86,7 @@ license = "Apache-2.0"
 publish = false
 repository = "https://github.com/rooch-network/rooch"
 rust-version = "1.82.0"
-version = "0.9.6"
+version = "0.9.7"
```

Comment:
Do not change the version, we should bump the version in a new release PR.
Change version to  0.9.6 and build again, rollback the change of openrpc.json.

Reply:
Or @jamesatomc you shoud submit the changed file 'sdk/typescript/rooch-sdk/src/version.ts'